internal/svcctl/comm.go

```diff
@@ -138,7 +138,7 @@ func HeartbeatHandler(cfg *config.Instance, resolver Resolver, analyticsReporter
 			}
 
 			logging.Debug("Firing runtime usage events for %s", metaData.Namespace)
-			analyticsReporter.Event(constants.CatRuntimeUsage, constants.ActRuntimeAttempt, dims)
+			analyticsReporter.Event(constants.CatRuntimeUsage, constants.ActRuntimeAttempt, constants.SrcExecutor, dims)
```

Comment:
It looks to me like all heartbeats come from executors, so that's why the source is this and not State Tool.